src/hotspot/share/jfr/leakprofiler/chains/bitset.cpp

```diff
@@ -29,6 +29,17 @@ BitSet::BitMapFragment::BitMapFragment(uintptr_t granule, BitMapFragment* next)
     _next(next) {
 }
 
+BitSet::BitMapFragmentTable::~BitMapFragmentTable() {
+    for (int index = 0; index < table_size(); index ++) {
+      Entry* e = bucket(index);
+      while (e != nullptr) {
+        Entry* tmp = e;
+        e = e->next();
+        free_entry(tmp);
+      }
+    }
```

Comment:
I think indenting is a bit off here: should be 2 spaces at all columns. Probably a copy-paste--from-destructor error.